scripts/main-yield-charts.main.v1.0.0.js

```diff
@@ -0,0 +1,257 @@
+const MOTNHS = ['Ene', 'Feb', 'Mar', 'Abr', 'May', 'Jun', 'Jul', 'Ago','Sept', 'Oct', 'Nov', 'Dic']
+var options = {
+  width: '100%',
+  height: 250,
+  colors: ['#25D16A'],
+  maintainAspectRatio: false,
+  legend: {
+    position:'none'
+  },
+  vAxis: {
+    format: "#.##'%'",
+    textStyle:{color: '#74797f'}
+  }, 
+  hAxis: {
+    textStyle:{color: '#74797f'}
+  },
+  pointSize: 5,
+  chartArea: {
+    height: '100%',
+    width: '100%',
+    top: 48,
+    left: 48,
+    right: 16,
+    bottom: 48
+  },
+  height: '100%',
+  width: '100%',
+};
+const numberToPercent = (number) => ((number - 1) * 100).toFixed(2) + '%';
+const mapAsDataArray = ({yields}) => yields.reduce((accData, yieldData) => {
+  const [year, month] = yieldData.mes.split('-');
+  const monthString = MOTNHS[Number(month) - 1];
+  return accData.concat([[`${monthString} ${year}`, yieldData.rentabilidadMes]])
+}, []);
+const getRentArrayWith = async ({year= '2023',month='01',risk=1,portfolio, typeUniverse}) => {
+  const url = `https://91o7sqo7s3.execute-api.us-east-1.amazonaws.com/prod/${portfolio}Yields`
+  var myHeaders = new Headers();
+  myHeaders.append("Content-Type", "application/json");
+  var raw = JSON.stringify({"year": year,"month": month,"risk": risk, typeUniverse});
+  var requestOptions = {
+    method: 'POST',
+    headers: myHeaders,
+    body: raw,
+    redirect: 'follow'
+  };
+  return fetch(url, requestOptions).then(response => response.json()).then(mapAsDataArray)
+};
+const mapExtractingPercentages = (dataArray) => dataArray.map(data => ([data[0], (data[1] - 1) * 100]))
```

Comment:
Qué viene en `data[0]` y `data[1]`? Puedes ponerlo como
```suggestion
const mapExtractingPercentages = (dataArray) => dataArray.map(([variable1, variable2]) => ([data[0], (data[1] - 1) * 100]))
```